conformance/ConformanceApi.fsd

```diff
@@ -108,7 +108,7 @@ service ConformanceApi
 	method getWidgetBatch
 	{
 		/// The IDs of the widgets to return.
-		[http(from: body), required]
+		[http(from: body), required, validate(count: ..100)]
```

Comment:
A violation will be easier to test if the maximum count is smaller, e.g. `..10`. (My bad.)

Reply:
On my machine I have a test that uses `Enumerable.Range(0, 101)`, but I can change it.
The big number and weird initialization doesn't make the test anymore true.